java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -690,6 +690,15 @@ public static ColumnVector concatenate(ColumnVector... columns) {
     }
   }
 
+  private static native long normalizeNANsAndZeros(long viewHandle) throws CudfException;
```

Comment:
nit: we have been placing the native declarations in a single group, but not very consistently.

Reply:
I thought that too. I wasn't sure where to place it. I'll move this to the end of the largest `native` block.

Reply:
Moved native method to more appropriate location.